src/components/dialogs/network-modifications/load/creation/load-creation-form.tsx

```diff
@@ -7,28 +7,29 @@
 
 import Grid from '@mui/material/Grid';
 import { EQUIPMENT_ID, EQUIPMENT_NAME, LOAD_TYPE, P0, Q0 } from 'components/utils/field-constants';
-import React from 'react';
-import {
-    ActivePowerAdornment,
-    filledTextField,
-    gridItem,
-    GridSection,
-    ReactivePowerAdornment,
-} from '../../../dialogUtils';
+import React, { FunctionComponent } from 'react';
+import { ActivePowerAdornment, gridItem, GridSection, ReactivePowerAdornment } from '../../../dialogUtils';
 import { LOAD_TYPES } from 'components/network/constants';
 import { FloatInput, SelectInput, TextInput } from '@gridsuite/commons-ui';
 import { ConnectivityForm } from '../../../connectivity/connectivity-form';
 import PropertiesForm from '../../common/properties/properties-form';
 import useVoltageLevelsListInfos from '../../../../../hooks/use-voltage-levels-list-infos';
+import { UUID } from 'crypto';
+import { CurrentTreeNode } from '../../../../../redux/reducer';
 
-const LoadCreationForm = ({ currentNode, studyUuid }) => {
+interface LoadCreationFormProps {
+    studyUuid: UUID;
+    currentNode: CurrentTreeNode;
+}
+
+const LoadCreationForm: FunctionComponent<LoadCreationFormProps> = ({ studyUuid, currentNode }) => {
     const voltageLevelOptions = useVoltageLevelsListInfos(studyUuid, currentNode?.id);
 
     const loadIdField = (
-        <TextInput name={EQUIPMENT_ID} label={'ID'} formProps={{ autoFocus: true, ...filledTextField }} />
+        <TextInput name={EQUIPMENT_ID} label={'ID'} formProps={{ autoFocus: true, variant: 'filled' }} />
```

Comment:
I think it was better to use `filledTextField` because it centralised the styles instead of copy pasting them.

Reply:
export const filledTextField = {
    variant: 'filled',
};
=> the same thing 
and the important that is not acceptable 
'filledTextField' does not exist in type 'Omit<TextFieldProps | TextFieldWithAdornmentProps, "onChange" | "value" | "inputRef" | "inputProps" | "InputProps">'.

Reply:
If this is an unsolvable type problem ok, no problem.

But copy and paste is not the same as using a constant. This reduces code maintenance.

Reply:
just typing filledTextField in the utility file